src/test/test-models/good/function-signatures/distributions/multivariate/discrete/ordered_logistic/ordered_logistic_log.stan

```diff
@@ -1,26 +1,42 @@
 data { 
   int d_int;
   real d_real;
+  int d_int_array[d_int];
   vector[d_int] d_vector;
+  vector[d_int] d_vector_array[d_int];
 }
 transformed data {
   real transformed_data_real;
 
   transformed_data_real <- ordered_logistic_log(d_int, d_real, d_vector);
+  transformed_data_real <- ordered_logistic_log(d_int_array, d_vector, d_vector);
+  transformed_data_real <- ordered_logistic_log(d_int_array, d_vector, d_vector_array);
```

Comment:
There isn't an int plus vector array version?  Most of our distributions allow free variation in what gets vectorized with other arguments being broadcast.  If that's not true here, ignore some of the other comments which were assuming this generality (the doc and signature definitions).

Reply:
As in: ```ordered_logistic_lpmf(int | double, vector[])```?
I hadn't thought to define that, I'll update the distributions in math to add those in